asabiris/handlers/webhandler.py

```diff
@@ -104,6 +104,9 @@ async def send_email(self, request, *, json_data):
 				attachments=json_data.get("attachments", []),  # Optional
 			)
 
+		except ValueError as e:
```

Comment:
Why ValueError is considered as "HTTPNotFound"?
That is behind this error?

Reply:
Then don't catch this here ... just pass that up. It's OK to produce 501 to the user in this situation.